runtime/doc/treesitter.txt

```diff
@@ -341,7 +341,7 @@ Lua module: vim.treesitter                               *lua-treesitter-core*
 get_parser({bufnr}, {lang}, {opts})                             *get_parser()*
                 Gets the parser for this bufnr / ft combination.
 
-                If needed this will create the parser. Unconditionnally attach
+                If needed this will create the parser. Unconditionally attach
```

Comment:
Typos from this line and below are generated from `runtime/lua/vim/treesitter.lua` and `runtime/lua/vim/treesitter/*` like from: https://github.com/neovim/neovim/blob/27da5511a0c0b12fcabe29cf38c3f8a0f0b444b9/runtime/lua/vim/treesitter.lua#L72

Reply:
:pray: